### DISCUSSION TITLE
rate limits handled?

### DISCUSSION THREAD

**Discussion by siccokoetsier** [xdevplatform/search-tweets-python#132] [Q&A]
Hi All,
I'm relatively new to python, twitter api, cloning projects on GitHub, your project and programming in general, yet I'm very enthusiastic about this project.
I do have an Academic Twitter track, but up to now have been unable to handle the rate limit errors generically (e.g. with Tweepy).
Therefore I pin my hope on searchtweet v2.
I just wanted to check with you: has this project dealt with the rate limit errors? Is it possible to collect large amounts (let's say +10k) of tweets since 2006? Has some sort of pagination/timing/sleep/safe api-requesting been provided for?
Thanks for your quick response, which will support me in my deep dive into this project :-).
KR

**Comment by igorbrigadir** [comment]
Yes - search-tweets-python will handle pagination and rate limit errors for you.
